addons/account/security/account_security.xml

```diff
@@ -7,23 +6,23 @@
 
         Invoicing only (account):
         ```
-         group_account_invoice ⇨ group_account_manager   (only those two should be used)
-                               ⬂
-                                 group_account_basic ⬂   (those are only visible through a server action)
-                              group_account_readonly ⇨ group_account_user
+         group_account_invoice &#8680; group_account_manager   (only those two should be used)
```

Comment:
Would it be possible to not remove the encoding prelude, and keep the arrows like before?